sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py

```diff
@@ -19,200 +19,214 @@
         except Exception as e:
             raise e
-
-        return r
-
-    def export_model(self, model_id, model_path="model.zip"):
-
-        # Export the model
-        model_stream_generator = self.ad_client.export_model(model_id)
-        with open(model_path, "wb") as f_obj:
-            while True:
-                try:
-                    f_obj.write(next(model_stream_generator))
-                except StopIteration:
-                    break
-                except Exception as e:
-                    raise e
+        
+        return None
 
     def delete_model(self, model_id):
 
         # Delete the model
         self.ad_client.delete_multivariate_model(model_id)
-        model_list_after_delete = list(self.ad_client.list_multivariate_model(skip=0, top=10000))
-        print("{:d} available models after deletion.".format(len(model_list_after_delete)))
+        model_list = self.list_models()
+        print("{:d} available models after deletion.".format(len(model_list)))
 
 
-    def last_detect(self, model_id, variables, detecting_points):
+    def last_detect(self, model_id, variables):
 
         # Detect anomaly by sync api
-        last_detection_req = LastDetectionRequest(variables=variables, detecting_points=detecting_points)
-        r = self.ad_client.last_detect_anomaly(model_id, last_detection_req)
+        r = self.ad_client.last_detect_anomaly(model_id, variables)
         print("Get last detection result")
         return r
 
 if __name__ == '__main__':
     SUBSCRIPTION_KEY = os.environ["ANOMALY_DETECTOR_KEY"]
     ANOMALY_DETECTOR_ENDPOINT = os.environ["ANOMALY_DETECTOR_ENDPOINT"]
 
-    # *****************************
-    # Use your own data source here
-    # *****************************
-    data_source = "<YOUR OWN DATA SOURCE>"
-
-    # Create a new sample and client
-    sample = MultivariateSample(SUBSCRIPTION_KEY, ANOMALY_DETECTOR_ENDPOINT, data_source)
+    ## Create a new sample and client
+    sample = MultivariateSample(SUBSCRIPTION_KEY, ANOMALY_DETECTOR_ENDPOINT)
 
     # Train a new model
-    model_id = sample.train(datetime(2021, 1, 1, 0, 0, 0, tzinfo=timezone.utc),
-                            datetime(2021, 1, 2, 12, 0, 0, tzinfo=timezone.utc))
-    assert model_id is not None
-
-    # Inference
-    result = sample.detect(model_id, datetime(2021, 1, 2, 12, 0, 0, tzinfo=timezone.utc),
-                           datetime(2021, 1, 3, 0, 0, 0, tzinfo=timezone.utc))
+    train_body = {
+        "slidingWindow": 200,
+        "alignPolicy": {
+            "alignMode": "Outer",
+            "fillNAMethod": "Linear",
+            "paddingValue": 0
+        },
+        "dataSource": "{blobUrl}",
+        "dataSchema": "MultiTable",
+        "startTime": "2021-01-02T00:00:00Z",
+        "endTime": "2021-01-02T05:00:00Z",
+        "displayName": "sample"
+    }
+    model_id = sample.train(train_body)
+
+    # Batch Inference
+    batch_inference_body = {
+        "dataSource": "{blobUrl}",
```

Comment:
the params 'startTime' and 'endTime' is related to sample data here. Will we provide a fixed sample data to user for them to try e2e successfully?